src/issuer/issuer.service.ts

```diff
@@ -326,15 +326,15 @@ export class IssuerService {
     /**
      * Revokes credential using the cred_rev_id and rev_reg_id
      */
-    public async revokeCredential(credential_exchange_id : string, publish : boolean): Promise<any> {
+    public async revokeCredential(credentialExchangeId : string, publish : boolean): Promise<any> {
```

Comment:
update "publish: boolean" to "publish=true"
Note that you no longer need to specify the type because it's implicit in the default value